batch/batch/cloud/gcp/instance_config.py

```diff
@@ -151,6 +153,7 @@ def from_dict(data: dict) -> 'GCPSlimInstanceConfig':
                 GCPDynamicSizedDiskResource('disk/pd-ssd/1'),
                 GCPIPFeeResource('service-fee/1'),
                 GCPServiceFeeResource('ip-fee/1024/1'),
+                GCPSupportLogsSpecsAndFirewallFees('gcp-support-logs-specs-and-firewall-fees/1'),
```

Comment:
I don't think you need this as it's for backwards compatibility. See comment in #13539.

Reply:
Can you elaborate on what kind of backwards compatibility? Should we remove this branch entirely?

Reply:
I think we can. It was for when the `instance_config` field was not in the database for old entries, so their value is NULL. We knew what the config should be at the time though. The only reason we'd need this code path is if we wanted to compute resources for old instances. I don't know why we would want to do that.

Reply:
https://github.com/hail-is/hail/pull/create?base=hail-is%3Amain&head=danking%3Aremove-unnecessary-backwards-compatibility-code&head_repo=hail